frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts

```diff
@@ -60,6 +60,7 @@ import {WorkPackageNotificationService} from "core-app/modules/work_packages/not
 import {combineLatest} from "rxjs";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {WorkPackagesTableComponent} from "core-components/wp-table/wp-table.component";
+import {GroupObject} from "core-app/modules/hal/resources/wp-collection-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e7ec4ddaf083fa89d159c00621383b6c27adf743/frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts#L64"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>